hal/armv7a/imx6ull/Makefile

```
@@ -4,5 +4,5 @@
 # Copyright 2021 Phoenix Systems
 #
 
-OBJS += $(addprefix $(PREFIX_O)hal/armv7a/imx6ull/, _init.o console.o imx6ull.o pmap.o timer.o interrupts.o)
+OBJS += $(addprefix $(PREFIX_O)hal/armv7a/imx6ull/, _init.o console.o imx6ull.o timer.o interrupts.o)
```

To consider adding a dependency here:
```suggestion
OBJS += $(addprefix $(PREFIX_O)hal/armv7a/imx6ull/, _init.o console.o imx6ull.o timer.o interrupts.o)

$(PREFIX_O)hal/armv7a/imx6ull/_init.o: hal/armv7a/_hal.S hal/armv7a/_interrupts.S
```
or
```suggestion
OBJS += $(addprefix $(PREFIX_O)hal/armv7a/imx6ull/, _init.o console.o imx6ull.o timer.o interrupts.o)

$(PREFIX_O)hal/armv7a/imx6ull/_init.o: $(PREFIX_O)hal/armv7a/imx6ull/_init.d
```
same for `zynq7000`

---

Done